### PR TITLE
first version

### PR DESCRIPTION
**Pull request by nadolinniy19**
- [DEMO LINK](https://nadolinniy19.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://nadolinniy19.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 42px;
+  padding: 12px 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.input::placeholder {
```
Comment (liliia-livandovskaya) [inline src/style.css:52]: add font-family: "Avenir", sans-serif; there

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form class="form search-box-big"
+    action="#" 
+    method="post"
+    data-qa="big"
+    >
```
Comment (liliia-livandovskaya) [inline src/index.html:19]:
```suggestion
    <form class="form search-box-big"
      action="#" 
      method="post"
      data-qa="big"
    >
```
pay attention to the code style

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,33 @@
...
+      <input
+      class="input input-big"
```
Comment (liliia-livandovskaya) [inline src/index.html:21]:
```suggestion
      class="input input-big" 
```
pay attention to the checklist

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
```
Comment (liliia-livandovskaya) [inline src/style.css:5]:
```suggestion
```

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: bold;
```
Comment (liliia-livandovskaya) [inline src/style.css:12]:
```suggestion
```

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 42px;
+  padding: 12px 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
```
Comment (liliia-livandovskaya) [inline src/style.css:50]: checklist p.2 [STYLES] - if you have two or more similar elements with portions of similar styles with different values - use one of the elements as the basic case, and override necessary styles for other cases. Explanation: The point is not in the names of the classes, the point is: when there are several similar elements, ex., 2 inputs, for one we can give a class input, for example, and for the second - input input--small. We write all the styles for .input, but for .input-small we write only those styles that differ in design, and we need this second input to look a little different. Element with class .input without extra classes should also look like a full-fledged styled element.

**Review by qoa11a — CHANGES_REQUESTED** [review]
image
You have failed tests. Looks like the problem is in the fonts.

**Comment by nadolinniy19** [comment]
Thanks for the tip about fonts. Everything works in the latest version.

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
```
Comment (Anastasiia-Svintsova) [inline src/style.css:113]: Delete comments

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
It seems that you still have problems with the fonts.

File: src/index.html [hunk 8]
```diff
@@ -11,6 +11,33 @@
...
+      method="post"
+      data-qa="small"
+    >
+      <input
+        class="input input-small" 
+        type="text"
+        name="search"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
```
Comment (stanislav-topikha) [inline src/index.html:40]:
```suggestion
    
    <form 
      class="form"
      action="#" 
      method="post"
      data-qa="big"
    >
      <input
        class="input" 
        type="text"
        name="search"
        placeholder="Try &quot;Los Angeles&quot;"
        data-qa="keypress"
      >
    </form>

    <form 
      class="form form--big"
      action="#" 
      method="post"
      data-qa="small"
    >
      <input
        class="input input-small" 
        type="text"
        name="search"
        placeholder="Try &quot;Los Angeles&quot;"
      >
    </form>
```
lets make it work like this

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]

**Comment by nadolinniy19** [comment]
Unfortunately couldn't use less classes

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
Great job 👏 
Left to fix second point of [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
no need in `--big` modifier. Use `big` styles for common selectors and redefine needed styles under `--small` selector

**Review by Bohdan-Maksymiuk — APPROVED** [review]
Good job 👍

**Pull request closed without merging**